modules/resource-metadata/main.tf

```diff
@@ -0,0 +1,204 @@
+locals {
+  function_name = "Coralogix-${var.package_name}-${random_string.this.result}"
+  coralogix_regions = {
+    Europe    = "ingress.coralogix.com:443"
+    Europe2   = "ingress.eu2.coralogix.com:443"
+    India     = "ingress.app.coralogix.in:443"
+    Singapore = "ingress.coralogixsg.com:443"
+    US        = "ingress.coralogix.us:443"
+    Custom    = var.custom_url
+  }
+  tags = {
+    Provider = "Coralogix"
+    License  = "Apache-2.0"
+  }
+}
+
+data "aws_region" "this" {}
+
+module "eventbridge" {
+  source = "terraform-aws-modules/eventbridge/aws"
+
+  create_bus = false
+
+  rules = {
+    crons = {
+      description         = "Trigger for a Lambda"
+      schedule_expression = var.schedule
+    }
+  }
+
+  targets = {
+    crons = [
+      {
+        name  = "cron-for-lambda"
```

Comment:
I would suggest appending a random string here like it is done for `function_name` to avoid name conflicts.

Reply:
i tested it and from what i saw i cant add random string to name in the eventbridge module, the module don't seem to support it. But i cant really think of a reason for a user to run the integration more then once in a region so there shouldn't be a conflicts.

Reply:
I see a reason to have it more than once in a region, that reason would be having multiple coralogix teams monitoring stuff deployed in the same aws account and region. But if it doesn't work then it doesn't work 🤷 .